src/robot/parsing/restreader.py

```diff
@@ -27,7 +27,10 @@ class RestReader(object):
         def read(self, rstfile, rawdata):
             doctree = publish_doctree(
                 rstfile.read(), source_path=rstfile.name,
-                settings_overrides={'input_encoding': 'UTF-8'})
+                settings_overrides={
+                    'input_encoding': 'UTF-8',
+                    'report_level': 4  # only severe messages
```

Comment:
Looking at docutils sources, it seems string `SEVERE` could be used instead of integer `4`. It would be easier to understand and remove the need for the comment.